index.js

```diff
@@ -15,13 +15,13 @@ class Client {
       throw new TypeError('Please provide auth options.')
     }
 
-    const { email, ethAddress, apiKey, password } = auth
+    const { email, ethAddress, apiKey, password, userId } = auth
 
-    if (!email && !ethAddress && !apiKey) {
+    if (!email && !ethAddress && !apiKey && !userId) {
       throw new TypeError('Please provide an user id auth option.')
     }
 
-    if (!apiKey && (!password && (email || ethAddress))) {
+    if (!apiKey && (!password && (email || ethAddress || userId))) {
```

Comment:
Same as above, `password` should be only required when there's no `apiKey` and there's `email` or `ethAddress`. Once we know that we are dealing with the trial user (no credentials given), we should not care if the password is missing for `userId`.